src/commands/LinkCommand.js

```diff
@@ -0,0 +1,30 @@
+import Command from "../Command";
+import PackageUtilities from "../PackageUtilities";
+
+export function handler(argv) {
+  return new LinkCommand(argv._, argv).run();
```

Comment:
Would you updating this to the new pattern [here](https://github.com/lerna/lerna/blob/4d64a2813ff0c35fd46ed745bbca1833e5cb86f0/src/commands/BootstrapCommand.js#L13-L14)?